Gruntfile.js

```diff
@@ -16,8 +16,18 @@ module.exports = function(grunt) {
           run: true
         }
       }
+    },
+    mocha_istanbul:{
+      coverage: {
+        src: 'test',
+        options:{
+          mask: '*-spec.js'
+        }
+      }
     }
   });
+  grunt.loadNpmTasks('grunt-mocha-istanbul');
```

Comment:
Doesn't `load-grunt-tasks` above in line 4 already load all the grunt- plugins?

I actually prefer having each module loaded explicitly, but just wanted to point this out.